tools/bin/commands/util/openshift_funcs

```diff
@@ -104,6 +104,31 @@ setup_oc() {
     exit 1
 }
 
+determine_console() {
+    local console=$(readopt --console)
+
+    if [ -z "${console}" ]; then
+        #
+        # 1. Show the status of oc
+        # 2. Cut the output to only the first line
+        # 3. Reduce the line to only the https ... url
+        #
+        # If logged out then console should be empty
+        #
+        console=$(oc status | head -n 1 | sed s/.*https/https/)
+    fi
+
+    if [[ ! ${console} =~ .*\/console ]]; then
+        #
+        # Ensure that console is appended on the end of the url
+        #
+        console="${console}/console"
```

Comment:
Shouldn't we always take over the console URL literally when provided from the outside ?

Reply:
Probably but then I never trust users to get it right first time ;-)
So checking for the console suffix seemed prudent.